openhtf/core/phase_executor.py

```diff
@@ -177,13 +177,14 @@ def execute_phases(self, phases, teardown_func):
 
     Args:
       phases: List of phases to execute.
-      teardown_func: 
+      teardown_func:
```

Comment:
teardown_func: Function or phase to execute at the end of all phases regardless of errors